multidimensional_urlencode/urlencoder.py

```diff
@@ -23,7 +26,7 @@ def flatten(d):
         return [[d]]
 
     returned = []
-    for key, value in d.items():
+    for key, value in list(d.items()):
```

Comment:
We need this because python3 returns a view of dictionarie's items. [source](https://docs.python.org/3/library/stdtypes.html#dict.items)